lib/pdk/module/build.rb

```diff
@@ -215,6 +213,8 @@ def ignored_files
               ignored = ignored.add("\/#{File.basename(target_dir)}\/")
             end
 
+            DEFAULT_IGNORED.each { |r| ignored.add(r) }
```

Comment:
This will add duplicate entries to the ignore list if there are common ignores between the pmt/pdk/git ignore files and the defaults.

Reply:
Duplicates shouldn't cause an issue (just as you can have duplicate entries in a `.gitignore`).

Reply:
True, we don't currently clear dupes from the ignore file either.